test/plugins/stats.js

```diff
@@ -0,0 +1,53 @@
+const request = require('supertest')
+const express = require('express')
+const {createRobot} = require('../..')
+const plugin = require('../../lib/plugins/stats')
+const nock = require('nock')
+
+nock.disableNetConnect()
+nock.enableNetConnect(/127\.0\.0\.1/)
+
+describe('stats', function () {
+  let robot, server
+
+  beforeEach(async () => {
+    nock('https://api.github.com')
+     .defaultReplyHeaders({'Content-Type': 'application/json'})
+     .get('/app/installations?per_page=100').reply(200, [{id: 1, account: {login: 'testing'}}])
+     .get('/installation/repositories').reply(200, {repositories: [
+       {private: true, stargazers_count: 1},
+       {private: false, stargazers_count: 2}
+     ]})
+
+    const cacheManager = require('cache-manager')
```

Comment:
any reason this shouldn't be declared at top of file?

Reply:
No, it should. I originally threw it here while I was trying to get the test passing because I wanted to refactor it out. A test shouldn't have to worry about setting up caching. I'll work on that before merging this.